diff --git cpp/Makefile.am cpp/Makefile.am
index b3d1a37..bd6a072 100644
--- cpp/Makefile.am
+++ cpp/Makefile.am
@@ -19,10 +19,10 @@
 #
 
 ossieName = RTL2832U
-bindir = $(prefix)/dev/devices/RTL2832U/cpp/
+bindir = $(SDR_ROOT)/dev/devices/RTL2832U/cpp/
 bin_PROGRAMS = RTL2832U
 
-xmldir = $(prefix)/dev/devices/RTL2832U/
+xmldir = $(SDR_ROOT)/dev/devices/RTL2832U/
 dist_xml_DATA = ../RTL2832U.scd.xml ../RTL2832U.prf.xml ../RTL2832U.spd.xml
 
 
