diff --git src/base/framework/Value.cpp src/base/framework/Value.cpp
index a347a48..8769c2e 100644
--- src/base/framework/Value.cpp
+++ src/base/framework/Value.cpp
@@ -176,12 +176,12 @@ const Value& ValueSequence::operator[] (size_t index) const
 
 void ValueSequence::push_back(const Value& value)
 {
-    ossie::corba::push_back(this, value);
+    ossie::corba::push_back(*this, value);
 }
 
 void ValueSequence::push_back(const CORBA::Any& value)
 {
-    ossie::corba::push_back(this, value);
+    ossie::corba::push_back(*this, value);
 }
 
 ValueSequence::iterator ValueSequence::begin()
diff --git src/base/framework/PropertyMap.cpp src/base/framework/PropertyMap.cpp
index d2c49e4..09d71a3 100644
--- src/base/framework/PropertyMap.cpp
+++ src/base/framework/PropertyMap.cpp
@@ -109,7 +109,7 @@ const Value& PropertyMap::operator[] (const std::string& id) const
 
 void PropertyMap::push_back(const CF::DataType& property)
 {
-    ossie::corba::push_back(this, property);
+    ossie::corba::push_back(*this, property);
 }
